### PR TITLE
Bugfix/1284

### PR DESCRIPTION
**Pull request by Elaine-Krauss-TCG**
#1284

### CODE REVIEW

**Comment by Elaine-Krauss-TCG** [comment]
There are E2E tests failing that are not failing on Develop
image
image

File: front-end/src/app/shared/components/transaction-type-base/transaction-type-base.component.ts [hunk 1]
```diff
@@ -122,120 +134,106 @@ export abstract class TransactionTypeBaseComponent implements OnInit, OnDestroy
...
+              subscriber.complete();
+            },
+          });
+        }).pipe(delay(500));
+      });
+
+    return from([of(true), ...confirmations$]).pipe(
+      concatAll(),
+      reduce((accumulator, confirmed) => accumulator && confirmed)
+    );
```
Comment (mjtravers) [inline front-end/src/app/shared/components/transaction-type-base/transaction-type-base.component.ts:216]: A great alternative to passing around the callbacks

**Review by mjtravers — DISMISSED** [review]

**Comment by mjtravers** [comment]
> There are E2E tests failing that are not failing on Develop image image

This is failing because it needs the patch already in develop branch

Oops, I spoke too soon. This is a legit bug. The navigation is failing on some transaction Save button clicks.

**Comment by Elaine-Krauss-TCG** [comment]
> 

I forgot that I made the PR, lol, so I can't approve it, but for what it matters I approve :+1:

**Review by toddlees — APPROVED** [review]

**Pull request merged**